cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/logging/Log.java

```diff
@@ -28,6 +29,8 @@
 
     // by default, we only want to log messages of levels info and greater
     public static final Level DEFAULT_LEVEL = INFO_LEVEL;
+    // in quiet mode, we only want to log errors
```

Comment:
You can set the modifier for to private and try to write a test for them.
Because you also have some tests for INFO_LEVEL and ERROR_Level.

Reply:
I agree that there should be one or more tests for this.

I wouldn't change the access modifier, though, we had some reason to leave this stuff public when first introducing the logging. Changing it for one item would be weird, changing them for all is worth a new PR.

Reply:
Added a test for the quiet mode, and improved the existing tests by adding a postcondition check, too.